addons/product/models/product_attribute.py

```diff
@@ -66,7 +66,7 @@ class ProductAttributeValue(models.Model):
     _order = 'attribute_id, sequence, id'
     _description = 'Attribute Value'
 
-    name = fields.Char(string='Value', required=True, translate=True)
+    name = fields.Char(string='Value', required=True, translate=True, index=True)
```

Comment:
As the field is translatable, is the index really efficient?
As the index is a BTree index, do you use startswith or equal comparison?